scripts/watch_txs.js

```diff
@@ -0,0 +1,783 @@
+
+        return from(
+          args[1].api.query.timestamp.now.at(args[0].block.block.header.hash)
+        ).pipe(
+          mapRx(
+            (timestamp) =>
+              `${msg} ${
+                signer ? `by <b>${signer}</b>` : "sent unsigned"
+              } at ${new Date(
+                +timestamp.toString()
+              ).toUTCString()} in ${buildExtrinsicUrl(
+                args[0].block.block.header.number,
+                args[0].rootTx?.hash
+              )}`
+          )
+        );
+      })
+    )
+  );
+
+/**
+ * Filter events produced by `democracy` pallet.
+ */
+const democracyEvent = eventByMethodFilter("democracy");
+/**
+ * Filter events produced by `council` pallet.
+ */
+const councilEvent = eventByMethodFilter("council");
+/**
+ * Filter events produced by `balances` pallet.
+ */
+const balancesEvent = eventByMethodFilter("balances");
+/**
+ * Filter events produced by `technicalCommittee` pallet.
+ */
+const technicalCommitteeEvent = eventByMethodFilter("technicalCommittee");
+/**
+ * Filter events produced by `technicalCommitteeMembership` pallet.
+ */
+const technicalCommitteeMembershipEvent = eventByMethodFilter(
+  "technicalCommitteeMembership"
+);
+/**
+ * Filter events produced by `elections` pallet.
+ */
+const electionsEvent = eventByMethodFilter("elections");
+/**
+ * Filter extrinsics from `democracy` pallet.
+ */
+const democracyTx = successfulTxByMethodFilter("democracy");
+/**
+ * Filter extrinsics from `balances` pallet.
+ */
+const balancesTx = successfulTxByMethodFilter("balances");
+/**
+ * Filter extrinsics from `system` pallet.
+ */
+const systemTx = successfulTxByMethodFilter("system");
+/**
+ * Filter extrinsics from `elections` pallet.
+ */
+const electionsTx = successfulTxByMethodFilter("elections");
+/**
+ * Filter extrinsics from `technicalCommitteeMembership` pallet.
+ */
+const technicalCommitteeMembershipTx = successfulTxByMethodFilter(
+  "technicalCommitteeMembership"
+);
+
+const txFilters = () => [
+  // Council candidacy submission
+  checkMap(electionsTx("submitCandidacy"), () =>
+    of(`Self-submitted as a council candidate`)
+  ),
+
+  // Council member removal
+  checkMap(
+    electionsTx("removeMember"),
+    ({
+      tx: {
+        args: [member],
+      },
+    }) => of(`Council member removed ${member.toString()}`)
+  ),
+
+  // `set_code` call
+  checkMap(systemTx("setCode"), () => of(`New runtime code was set`)),
+
+  // `set_code_without_checks` call
+  checkMap(systemTx("setCodeWithoutChecks"), () =>
+    of(`New runtime code was set without checks`)
+  ),
+
+  // `set_storage` call
+  checkMap(systemTx("setStorage"), () => of(`Storage was modified directly`)),
+
+  // Democracy proposal seconded
+  checkMap(
+    democracyTx("second"),
+    ({
+      tx: {
+        args: [idx],
+      },
+    }) => of(`Proposal #${idx.toString()} is seconded`)
+  ),
+
+  // `force_transfer` call
+  checkMap(balancesTx("forceTransfer"), ({ tx }) => {
```

Comment:
Should also check for `setBalance`

Reply:
https://github.com/docknetwork/sdk/pull/304/files#diff-87900fe8bf8aecbf596084810327d179ed7f4c61111e5ce2273449b4120d95aeR569

Reply:
Did you mean to redirect to the `BalanceSet` event?

Reply:
Right, just something wrong with the link
https://github.com/docknetwork/sdk/pull/304/files#diff-87900fe8bf8aecbf596084810327d179ed7f4c61111e5ce2273449b4120d95aeR586

Reply:
Thanks.